.github/workflows/task-build-artifacts.yml

```diff
@@ -117,8 +101,25 @@ jobs:
                   ;;
               esac
 
-              # Perform checkout to the REF
-              perform_checkout_ref
+              # Configure the safe directory
```

Comment:
Not part of this PR, right?

Reply:
Workflows were not modified by this PR. You saw it during in the review because I merged master branch in this branch